conf/stats.conf.example

```diff
@@ -1,102 +1,97 @@
+type=mysql_query
+statsd_type=histogram
+statsd_ns=source.packetfence.devices.registered_per_role
+mysql_query=select name, count(distinct node.mac) from node_category right join node on node.category_id=node_category.category_id where node.status='reg' group by node_category.category_id;
+interval=60s
+
+[metric 'total number of unregistered devices']
+type=mysql_query
+statsd_type=histogram
+statsd_ns=source.packetfence.devices.unregistered
+mysql_query=select count(distinct mac) from node where status='unreg'
+interval=60s
 
 [metric 'total online devices']
 type=mysql_query
 statsd_type=histogram
 statsd_ns=source.packetfence.devices.online
-# mysql_query must return a single column with a single row
 mysql_query=select count(distinct callingstationid) from radacct where acctstoptime is null
 interval=60s
 
 [metric 'total offline devices']
 type=mysql_query
 statsd_type=histogram
 statsd_ns=source.packetfence.devices.offline
-# mysql_query must return a single column with a single row
 mysql_query=select count(distinct callingstationid) from radacct where acctstoptime <= NOW()
 interval=60s
 
 [metric 'number of new registered devices during the past hour']
 type=mysql_query
 statsd_type=histogram
 statsd_ns=source.packetfence.devices.registered_last_hour
-# mysql_query must return a single column with a single row
 mysql_query=select count(distinct mac) from node where status='reg' and regdate >= NOW() - INTERVAL 1 HOUR
 interval=60s
 
 [metric 'number of new registered devices during the past day']
 type=mysql_query
 statsd_type=histogram
 statsd_ns=source.packetfence.devices.registered_last_day
-# mysql_query must return a single column with a single row
 mysql_query=select count(distinct mac) from node where status='reg' and regdate >= NOW() - INTERVAL 1 DAY
 interval=60s
 
 [metric 'number of new registered devices during the past week']
 type=mysql_query
 statsd_type=histogram
 statsd_ns=source.packetfence.devices.registered_last_week
-# mysql_query must return a single column with a single row
 mysql_query=select count(distinct mac) from node where status='reg' and regdate >= NOW() - INTERVAL 1 WEEK
 interval=60s
 
 [metric 'number of new registered devices during the past month']
 type=mysql_query
 statsd_type=histogram
 statsd_ns=source.packetfence.devices.registered_last_month
-# mysql_query must return a single column with a single row
 mysql_query=select count(distinct mac) from node where status='reg' and regdate >= NOW() - INTERVAL 1 MONTH
 interval=60s
 
 [metric 'number of new registered devices during the past year']
 type=mysql_query
 statsd_type=histogram
 statsd_ns=source.packetfence.devices.registered_last_year
-# mysql_query must return a single column with a single row
 mysql_query=select count(distinct mac) from node where status='reg' and regdate >= NOW() - INTERVAL 1 YEAR
 interval=60s
 
 [metric 'number of devices currently online and in the registration network']
 type=mysql_query
 statsd_type=histogram
```

Comment:
i think you see the pattern now :P

They are pretty much all gauges when you're not incrementing something or calculating a timing